libc-test/build.rs

```diff
@@ -1927,6 +1928,157 @@ fn test_emscripten(target: &str) {
     cfg.generate("../src/lib.rs", "main.rs");
 }
 
+fn test_vxworks(target: &str) {
+    assert!(target.contains("vxworks"));
+
+    let mut cfg = ctest::TestGenerator::new();
+    headers! { cfg:
+               "vxWorks.h",
+               "yvals.h",
+               "nfs/nfsCommon.h",
+               "rtpLibCommon.h",
+               "randomNumGen.h",
+               "taskLib.h",
+               "sysLib.h",
+               "ioLib.h",
+               "inetLib.h",
+               "socket.h",
+               "errnoLib.h",
+               "ctype.h",
+               "dirent.h",
+               "dlfcn.h",
+               "elf.h",
+               "fcntl.h",
+               "grp.h",
+               "sys/poll.h",
+               "ifaddrs.h",
+               "langinfo.h",
+               "limits.h",
+               "link.h",
+               "locale.h",
+               "sys/stat.h",
+               "netdb.h",
+               "pthread.h",
+               "pwd.h",
+               "sched.h",
+               "semaphore.h",
+               "signal.h",
+               "stddef.h",
+               "stdint.h",
+               "stdio.h",
+               "stdlib.h",
+               "string.h",
+               "sys/file.h",
+               "sys/ioctl.h",
+               "sys/socket.h",
+               "sys/time.h",
+               "sys/times.h",
+               "sys/types.h",
+               "sys/uio.h",
+               "sys/un.h",
+               "sys/utsname.h",
+               "sys/wait.h",
+               "netinet/tcp.h",
+               "syslog.h",
+               "termios.h",
+               "time.h",
+               "ucontext.h",
+               "unistd.h",
+               "utime.h",
+               "wchar.h",
+               "errno.h",
+    }
+
+    cfg.skip_struct(move |ty| {
+        match ty {
+            "passwd"      |
+            //the below definitions is part of public/private/ 
+            "_Vx_EVENTS_RSRC" |
+            "_Vx_semaphore" |
+            //Avaliable in Kernel space.
+            "epoll_event" => true,
+            _ => false,
+        }
+    });
+
+    /* Fix me */
+    cfg.skip_const(move |name| match name {
+        "SIG_DFL"
+        | "SIG_IGN"
+        | "SIG_ERR"
+        | "EPOLLPRI"
+        | "EPOLLIN"
+        | "EPOLLOUT"
+        | "EPOLLERR"
+        | "EPOLLHUP"
+        | "EPOLLRDHUP"
+        | "EPOLLONESHOT"
+        | "EPOLLET"
+        | "EPOLL_CTL_ADD"
+        | "EPOLL_CTL_DEL"
+        | "EPOLL_CTL_MOD"
+        | "S_nfsLib_NFSERR_XDEV"
+        | "S_nfsLib_NFSERR_NODEV"
+        | "S_nfsLib_NFSERR_MLINK"
+        | "S_IFDEVMEM"
+        | "CHILD"
+        | "TIME_RELTIME"
+        | "RTLD_DEFAULT"
+        | "EIOA"
+        | "FIONBIO"
+        | "FIOWRITE"
+        | "FIONREAD"
+        | "VXSIM_EWOULDBLOCK" => true,
+        _ => false,
+    });
+
+    /* Fix me */
```

Comment:
'Fix me' - Is this a result of copy-paste or is there something that needs to be resolved here?

Reply:
The code was actually developed by Umesh. I believe "Fix me" means we don't know how to test them yet so we skip them right now. We will remove them from the "skip list" when we know how to fix it.

Reply:
like @BaoshanPang mentioned, we  will refactor it going forward.